config.rb.sample

```diff
@@ -46,3 +46,6 @@
 #$vb_gui = false
 #$vb_memory = 1024
 #$vb_cpus = 1
+
+# Share additional folders to the CoreOS VMs
```

Comment:
Could you document the example in a comment, leaving the default behavior documented below? For example:

```
# Share additional folders to the CoreOS VMs. For example, $shared_folders = ['~src/app1', '~/src/app2']
#$shared_folders = []
```